apps/stats/src/views/Stats/Web.tsx

```
@@ -202,31 +231,102 @@ const TopContentTable: React.FC<TopContentTableProps> = ({data}) => {
+        }
+        
+        // Filter content data based on type
+        let filteredData: TopContentData[];
+        switch (selectedContentType) {
+        case CONTENT_TYPES.POSTS:
+            filteredData = data.filter(item => item.post_uuid && item.post_uuid.trim() !== '');
+            break;
+        case CONTENT_TYPES.PAGES:
+            filteredData = data.filter(item => !item.post_uuid || item.post_uuid.trim() === '');
+            break;
+        case CONTENT_TYPES.POSTS_AND_PAGES:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)